src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -346,10 +244,16 @@ function getArrayDataPointer<T>(array: System_Array<T>): number {
   return <number><any>array + 12; // First byte from here is length, then following bytes are entries
 }
 
+function bindStaticMethod(assembly: string, typeName: string, method: string) : (...args: any[]) => any {
```

Comment:
Shouldn't this be typed as returning `(...args: (System_Object | null)[]) => System_Object` and not `(...args: any[]) => any`?

I suspect it would be good to declare a type referencing a bound method. For example, in MonoType.d.ts, add something like:

```ts
type BoundStaticMethod = (...args: (System_Object | number | null)[]) => (System_Object | number | null);
```

... and then `mono_bind_static_method` can be declared as:

```
function mono_bind_static_method(fqn: string): BoundStaticMethod;
```

... and then any other uses of it can reference `BoundStaticMethod` as appropriate too.